backend/server.ts

```diff
@@ -27,6 +27,10 @@ app.use((_req, res, next) => {
 
 app.use(express.static(path.join(dirName, './dist')));
 
+app.get('/.well-known/assetlinks.json', (_req: Request, res: Response) => {
```

Comment:
How do you feel about adding a link to https://developer.android.com/training/app-links/verify-android-applinks#publish-json here as well?  Also, a note about why we're adding it (i.e., a "tl;dr" version of the link) would be helpful to our users, IMO.